src/test/java/org/isf/dicom/Tests.java

```diff
@@ -343,16 +343,16 @@ public void testFileDicomHashCode() throws Exception {
 	}
 
 	@Test
-	public void testFileDicomGetDicomType() throws Exception {
+	void testFileDicomGetDicomType() throws Exception {
 		DicomType dicomType = testDicomType.setup(true);
 		FileDicom fileDicom = testFileDicom.setup(dicomType, true);
 		assertThat(fileDicom.getDicomType()).isEqualTo(dicomType);
 	}
 
 	// This test requires access to the opencv native libraries
-	@Ignore
+	@Disabled
```

Comment:
Should we give a try to all these `@Disabled` tests?

Reply:
Seems like it runs cleanly now.